dotnet-projects/StyleCop/StyleCop/GlobalSuppressions.cs

```diff
@@ -0,0 +1,7 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
```

Comment:
In the article there is a line that is added to this file, I don't see it here. 
Once you add it please break it into multiple lines as well (after each comma). 
And update the code in the article.